src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String COLOR = Color.WHITE.name();
+    private static final int CIRCLE_DEFAULT_RADIUS = 10;
+    private static final int MAX_VALUE = 30;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure[] figures = new Figure[]{
+                new Circle(colorSupplier.getRandomColor(), getRandomValue()),
+                new Square(colorSupplier.getRandomColor(), getRandomValue()),
+                new Rectangle(colorSupplier.getRandomColor(), getRandomValue(),
+                        getRandomValue()),
+                new RightTriangle(colorSupplier.getRandomColor(), getRandomValue(),
+                        getRandomValue()),
+                new IsoscelesTrapezoid(colorSupplier.getRandomColor(), getRandomValue(),
+                        getRandomValue(), getRandomValue())
+        };
+        return figures[random.nextInt(figures.length)];
+    }
```

Comment:
I don't like solution where we create 5 different objects each time method is called when we require just 1 of them.
Please redesign method to create only required type of figure